networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc

```diff
@@ -38,6 +38,18 @@ include::modules/virt-removing-interface-from-nodes.adoc[leveloffset=+2]
 [id="virt-nmstate-example-policy-configurations"]
 == Example policy configurations for different interfaces
 
+The following examples show different `NodeNetworkConfigurationPolicy` manifest configurations.
+
+For best performance, consider the following factors when applying a policy:
+
+* When you need to apply a policy to more than one node, create a `NodeNetworkConfigurationPolicy` manifest for each target node. Scoping a policy to a single node reduces the overall length of time for the NMState Operator to apply the policies. 
++
+In contrast, if a single policy includes configurations for several nodes, the NMState Operator applies the policy to each node in sequence, which increases the overall length of time for policy application. 
+
+* All related network configurations should be specified in a single policy. 
++
+When a node reboots, the NMState Operator cannot control the order in which policies are applied. Therefore, the NMState Operator might apply interdependent policies in a sequence that results in a degraded network object. 
```

Comment:
Same comment 
[Usage of "reboot"](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#r): Reboot is widely recognized among technical users, but where usage permits, use restart.